packages/@webex/plugin-meetings/src/config.ts

```diff
@@ -92,5 +92,6 @@ export default {
     degradationPreferences: {
       maxMacroblocksLimit: 8192,
     },
+    iceCandidatesGatheringTimeout: undefined,
```

Comment:
maybe worth adding a comment that this only applies to non-multistream meetings?

Reply:
Done